src/handlers/sync-labels-to-config.ts

```diff
@@ -23,6 +22,48 @@ export async function syncPriceLabelsToConfig(context: Context) {
   // List all the labels for a repository
   const allLabels = await listLabelsForRepo(context);
 
+  const incorrectPriceLabels = allLabels.filter((label) => label.name.startsWith("Price: ") && !priceLabels.includes(label.name));
+
+  if (incorrectPriceLabels.length > 0 && config.globalConfigUpdate) {
+    logger.info("Incorrect price labels found, removing them", { incorrectPriceLabels: incorrectPriceLabels.map((label) => label.name) });
+    const owner = context.payload.repository.owner?.login;
+    if (!owner) {
+      throw logger.error("No owner found in the repository!");
+    }
+    await Promise.allSettled(
```

Comment:
Doesn't the rest halt execution if any fail in the chain of requests? Maybe we should loop and do each request individually.